packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -80,11 +122,49 @@ h1 {
   margin: 0.166em 0.33em 0.33em;
 }
 
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip .tooltiptext {
+  opacity: 0.4;
+  visibility: hidden;
+  width: 200px;
+  font-size: 0.8em;
+  background-color: black;
+  color: #fff;
+  text-align: left;
+  border-radius: 6px;
+  padding: 5px 5px;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+}
+
+.question-icon{
+  background-image: url(../../shared-assets/icons/help_outline-24px.svg);
+  background-size : 0.8em;
+  background-repeat: no-repeat;
+  background-position: 0% 100%;
+  width: 1em;
+  height: 1em;
+  display: inline-block;
+}
+
+.metrics{
+  text-align: center;
+  height: 2em;
+}
+
```

Comment:
im really not good at css, so im not sure whether these styles are written in the best way

Reply:
Me neither; I was going to ask if you've tested it on mobile, but I just tried the current version and it's kind of a disaster.